concent_api/gatekeeper/views.py

```diff
@@ -35,9 +36,13 @@
 @require_POST
 def upload(request):
     logger.debug("Upload request received.")
+
     if request.content_type in ['multipart/form-data', '', None] or request.content_type.isspace():
         logger.info(f'Unsupported content type: {request.content_type}')
-        return gatekeeper_access_denied_response('Unsupported content type.')
+        return gatekeeper_access_denied_response(
+            'Unsupported content type.',
+            ErrorCode.HEADER_CONTENT_TYPE_NOT_SUPPORTED.value,
```

Comment:
That's not part of this pull request but maybe we could change it anyway: this should be HTTP 415.

Reply:
:+1:

Reply:
Sorry for adding you extra work but I was wrong here. It can't be HTTP 415. Normally it should, but the `auth_request` module in nginx can only handle HTTP 401. So please change it back. The status code parameter can stay. Just change the number.

Could you also add a comment explaining it so that this does not happen again? It's not obvious and now I fell into this trap myself even though I know about it :)

Reply:
:+1: done